plugins/smssync/classes/Controller/Sms/Smssync.php

```diff
@@ -91,7 +98,7 @@ private function _incoming()
 		// Remove Non-Numeric characters because that's what the DB has
 		$to = preg_replace("/[^0-9,.]/", "", $this->request->post('sent_to'));
 
-		$from = preg_replace("/[^0-9,.]/", "", $from);
+		$from = preg_replace("/[^0-9A-Za-z,.]/", "", $from);
```

Comment:
Maybe we should limit this to alphanumeric senders? i.e 1 - 11 alphanumeric characters including a space.